app/config/services/twig.yml

```diff
@@ -5,3 +5,7 @@ services:
             - '@ezpublish.api.service.content_type'
         tags:
             - name: twig.extension
+    twig.extension.text:
```

Comment:
This is required for Twig `truncate` function which is used here: [https://github.com/ezsystems/ezplatform-com/pull/101/files#diff-205461f0c399ee7cdb4ed7a7eda49f76R36](https://github.com/ezsystems/ezplatform-com/pull/101/files#diff-205461f0c399ee7cdb4ed7a7eda49f76R36)